Tests/AppTests/AppTestCase.swift

```diff
@@ -64,9 +58,12 @@ extension AppTestCase {
         var serializer = SQLSerializer(database: app.db as! SQLDatabase)
         query?.serialize(to: &serializer)
         return serializer.binds.reduce(into: []) { result, bind in
-            if let bind = bind as? String { result.append(bind) }
-            if let bind = bind as? Date { result.append(DateFormatter.filterParseFormatter.string(from: bind)) }
-            if let bind = bind as? Int { result.append(String(bind)) }
+            switch bind {
+                case let bind as Date:
+                    result.append(DateFormatter.filterParseFormatter.string(from: bind))
+                default:
+                    result.append("\(bind)")
+            }
```

Comment:
There was a bug here where a new type (i.e. `Set<PlatformCompatibility>`) would be skipped from the bind variables. This lead me to debugging binds and ending down the rabbit hole with SQLBind and quoting...

Now we just default to `"\(bind)"` as a replacement which will always yield _something_ and we can then specialise it with a new case above.